internal/clients/image_builder/image_client.go

```diff
@@ -20,13 +20,36 @@ func init() {
 }
 
 func newImageBuilderClient(ctx context.Context) (clients.ImageBuilder, error) {
-	c, err := NewClientWithResponses(config.ImageBuilder.URL)
+	c, err := NewClientWithResponses(config.ImageBuilder.URL, func(c *Client) error {
+		if config.ImageBuilder.Proxy.URL != "" {
```

Comment:
It makes me bit unease to not have gate for this by not being in prod environment :thinking:

Reply:
So let me translate: you would like to have a prod gating? I can do that.

Reply:
yeah, I know how to write unreadable sentences, don't I xD
That was exactly what I meat, thanks :+1: